treeherder/webapp/api/serializers.py

```diff
@@ -185,19 +185,7 @@ def to_representation(self, failure_line):
 class TextLogErrorSerializer(serializers.ModelSerializer):
     class Meta:
         model = models.TextLogError
-        exclude = ["step"]
-
-
-class TextLogStepSerializer(serializers.ModelSerializer):
-    errors = TextLogErrorSerializer(many=True, read_only=True)
-    result = serializers.SerializerMethodField()
-
-    def get_result(self, obj):
-        return obj.get_result_display()
-
-    class Meta:
-        model = models.TextLogStep
-        exclude = ["job"]
+        exclude = []
```

Comment:
This line can be removed.

Reply:
this is removed and replaced with `fields = "all"` as per modern django (3.30+) requirements for serializers.

As I had to reclone my repo, I ended up with a new PR: https://github.com/mozilla/treeherder/pull/8577